app/js/reducers/encounterRoleReducer.js

```diff
@@ -0,0 +1,41 @@
+import {
+  FETCH_ENCOUNTER_ROLE_LOADING,
+  FETCH_ENCOUNTER_ROLE_SUCCESS,
+  FETCH_ENCOUNTER_ROLE_FAILURE,
+} from '../../../app/js/actions/actionTypes';
+
+const initialState = {
+  isLoading: false,
+  encounterRole: {},
+  error: null,
+};
+
+const encounterRole = (state = initialState, action) => {
+  switch (action.type) {
+    case FETCH_ENCOUNTER_ROLE_LOADING:
+      return {
+        ...state,
+        isLoading: action.status,
+        encounterRole: {},
+        error: null,
```

Comment:
Lines 19 and 20 are kinda unnecessary because those values are already available as default in the initialState, so I guess it's safe to remove them. Thanks